halyard/hal-scripts/spin-gate-api.sh

```diff
@@ -27,50 +27,60 @@ echo "Added Gate x509 API Service for deployment named ${deployment}"
 
 echo "Creating Gate API Service for deployment named ${deployment}"
 cat <<SVC_EOF | kubectl --kubeconfig="${details.kubeConfig}" apply -f -
-apiVersion: v1
-kind: Service
+apiVersion: extensions/v1beta1
+kind: Ingress
 metadata:
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/backend-protocol: HTTPS
+    nginx.ingress.kubernetes.io/ssl-passthrough: "true"
+    nginx.ingress.kubernetes.io/ssl-redirect: "true"
   labels:
     app: spin
     cluster: spin-gate
   name: spin-gate-api
   namespace: spinnaker
 spec:
-  loadBalancerIP: ${details.gateApiIP}
-  ports:
-  - port: 443
-    protocol: TCP
-    targetPort: 8084
-  selector:
-    app: spin
-    cluster: spin-gate
-  sessionAffinity: None
-  type: LoadBalancer
+  rules:
+  - host: ${details.gateApiHostname}
+    http:
+      paths:
+      - backend:
+          serviceName: spin-gate
+          servicePort: 8084
+        path: /
+status:
```

Comment:
Please remove the `status:` stanza and associated child.

Reply:
fixed in ca402de